src/ToDo.UI/Extensions/NameValueCollectionExtensions.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Specialized;
+
+namespace ToDo.Extensions
+{
+    public static class NameValueCollectionExtensions
```

Comment:
This NameValueCollectionExtensions is used for what exactly for the app?

Reply:
in App.xaml.host.cs as NameValueCollection doesn't have a tryget

Reply:
dont really need that